examples/Demo/FluentUI.Demo.Client/Documentation/Components/CompoundButton/FluentCompoundButton.md

```diff
@@ -0,0 +1,29 @@
+---
+title: CompoundButton
+route: /CompoundButton 
+---
+
+# CompoundButton
+
+## Default
+
+{{ CompoundButtonDefault }}
```

Comment:
I don't think it's necessary to explain: a Blazor developer already knows how to use a `Label` parameter or a `ChildContent` parameter. The description of `Label` is sufficient to explain the difference between `ChildContent` and `Label`.

I would remove the Default.razor file, which seems to provide nothing.